src/main/java/com/mraof/minestuck/blockentity/TransportalizerBlockEntity.java

```diff
@@ -167,9 +177,19 @@ protected void teleport(Entity entity)
 				return;
 			}
 			
-			entity = Teleport.teleportEntity(entity, (ServerLevel) destTransportalizer.level, destination.pos().getX() + 0.5, destination.pos().getY() + 0.6, destination.pos().getZ() + 0.5, entity.getYRot(), entity.getXRot());
+			ServerLevel originServerLevel = (ServerLevel) this.level;
+			ServerLevel destinationServerLevel = (ServerLevel) destTransportalizer.level;
```

Comment:
Just `originLevel` and `destinationLevel` should be enough.

Reply:
Reorganized